controllers/accountControllers/userAccountController.js

```diff
@@ -0,0 +1,132 @@
+const {Account} = require('../../models/userAccountModel');
+const {User} = require('../../models/userModel');
+const {validateAccount} = require('../../validation/validationAccount');
+
+
+const createAccount =  async (req, res, next) => {
+    const { error } = await validateAccount.validate(req.body);
+    if(error){
+        return next (new Error(error));
+    }
+    const {
+        accountName, 
+        accountCurrency, 
+        accountAmount, 
+        accountDescription, 
+        accountStat
+    }   = req.body;
+    
+    if (
+        accountName && 
+        accountCurrency && 
+        accountAmount && 
+        accountDescription && 
+        accountStat
+        ) {
+
+        const isAccount = await Account.findOne({accountCurrency});
```

Comment:
Please explain this logic

Reply:
When a user is creating a new currency account, it checks whether such an account name has already been created or not. My mistake here is that I checked the currency of the account for instead of the account name